content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -325,13 +325,13 @@ Name:      kubernetes.default
 Address 1: 10.0.0.1 kubernetes.default.svc.cluster.local
 ```
 
-If this fails, you might need to go to the kube-proxy section of this doc, or
-even go back to the top of this document and start over, but instead of
+If this fails, you might need to go to the kube-proxy section of this document,
+or even go back to the top of this document and start over, but instead of
```

Comment:
You could add a link to the kube-proxy section in this page:
Line 328, 
If you are unable to communitcate with the Kubernetes master Service, see the [kube-proxy](#Is-the-kube-proxy-working) section of this document, or even ...